README.md

```diff
@@ -448,3 +448,20 @@ Ensures all values are integers or floats.
 * `gt`, `gte`, `lt`, and `lte` are respectively greater than, greater than or equal,
```

Comment:
For the `.gitignore` and `sdk.cfg` files, just deleting them isn't going to do the trick, since they'll still be tracked. They need to be removed from the history of this PR.